infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryServiceAdapter.java

```diff
@@ -18,7 +22,7 @@ public interface CorfuReplicationDiscoveryServiceAdapter {
      *
      * @return
      */
-    Map<String, LogReplicationMetadata.ReplicationStatusVal> queryReplicationStatus();
+    Map<LogReplication.LogReplicationSession, LogReplicationMetadata.ReplicationStatus> queryReplicationStatus();
```

Comment:
If this interface is exposed to client, should we create a new endpoint? Otherwise, client changes would be needed. Need to add a TODO in the later case.